libs/ui/src/lib/portfolio-proportion-chart/portfolio-proportion-chart.component.ts

```diff
@@ -77,7 +77,7 @@ export class GfPortfolioProportionChartComponent
 
   @ViewChild('chartCanvas') chartCanvas: ElementRef<HTMLCanvasElement>;
 
-  public chart: Chart<'pie'>;
+  public chart: Chart<'doughnut'>;
```

Comment:
👀

Reply:
Yeah, I believe there's a discrepancy in the chart type all this time. The chart type is declared as 'pie' here but initialized as 'doughnut' on line 362 😄